libs/core/src/lib/pagination/pagination.component.scss

```diff
@@ -10,3 +10,7 @@ $block: fd-pagination;
         }
     }
 }
+
+.cdk-live-announcer-element {
```

Comment:
why do we need this? announcer's elements are not visible

And even if we do, this style will start working once pagination component is loaded, affecting the entire app. We should not have global styles like this

Reply:
Yea, you right, thanks, fixed it. 

Anyway, it visible in stackblitz examples, so i kept this rule only moved in the different file.

Reply:
@platon-rov I've noticed that now messages won't be announced by screen reader. Maybe we should have something else instead of `display: none`? We have `screenreader-only` style in a few places, maybe we should reuse it and apply here?
```
.screenreader-only {
    position: absolute !important; /* Outside the DOM flow */
    height: 1px;
    width: 1px; /* Nearly collapsed */
    overflow: hidden;
    opacity: 0;
    clip: rect(1px 1px 1px 1px); /* IE 7+ only support clip without commas */
    clip: rect(1px, 1px, 1px, 1px); /* All other browsers */
}
```

Reply:
Good catch, thank you, please check :)

Reply:
I've added these styles to the stackblitz styles file as only there we see this live announcer text, in our apps we don't have it.

Reply:
Also checked in external app, text reads & element isn't visible.